source/com.microsoft.tfs.client.eclipse.ui.egit/src/com/microsoft/tfs/client/eclipse/ui/egit/importwizard/CrossCollectionRepositorySelectControl.java

```diff
@@ -203,26 +208,71 @@ public void run() {
     }
 
     private String getDefaultGitRootFolder() {
-        final IScopeContext scope = new InstanceScope();
-        final IEclipsePreferences prefs = scope.getNode(EGIT_PREF_STORE_ID);
+        final String rootFolderPreference = getDefaultRootFolderPreference();
+        Check.notNull(rootFolderPreference, "Default root folder preference"); //$NON-NLS-1$
 
-        Check.notNull(prefs, "Egit preferences store"); //$NON-NLS-1$
+        // The preference could contain variables. Let's expand all of them.
+        final String rootDirectory = expandVariables(rootFolderPreference);
```

Comment:
expandVariables may get an exception trying to expand and then return null. I think expandVariables should return the original string if it fails to expand for any reason. (see comment below)